website/content/en/docs/olm-integration/legacy/generating-a-csv.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Generating Manifests and Metadata
+linkTitle: Generating Manifests and Metadata
+weight: 20
+---
+
+This document describes how to manage packaging and shipping your Operator in the following stages:
+
+* **Generate your first release** - encapsulate the metadata needed to install your Operator with the
+[Operator Lifecycle Manager][olm] and configure the permissions it needs from the generated SDK files.
+* **Update your Operator** - apply any updates to Operator manifests made during development.
+* **Upgrade your Operator** - carry over any customizations you have made and ensure a rolling update to the
+next version of your Operator.
+
+## Overview
+
+Several `operator-sdk` subcommands manage operator-framework manifests, in particular [`ClusterServiceVersion`'s (CSVs)][doc-csv],
+for an Operator: [`generate bundle`][cli-gen-bundle] and [`generate packagemanifests`][cli-gen-packagemanifests].
+See this [CLI overview][cli-overview] for details on each command.
+
+### ClusterServiceVersion manifests
+
+CSV's are manifests that define all aspects of an Operator, from what CRDs it uses to metadata describing the
```

Comment:
Nit: this should be plural instead of possessive.

```suggestion
CSVs are manifests that define all aspects of an Operator, from what CRDs it uses to metadata describing the
```